backend/server.js

```diff
@@ -4,6 +4,8 @@
 // init project
 const express = require('express');
 const path = require('path');
+const bodyParser = require('body-parser');
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
```

Comment:
I don't think we can use env vars before `require('dotenv').config();`. When I tested I had to move line 12 up before line 8.

Reply:
Mh, actually, since we already have it in the dev.js file, we should be able to remove here.